src/Sentry.Unity.Editor/SentryWindow.cs

```diff
@@ -63,85 +93,250 @@ private void CheckForAndConvertJsonConfig()
+                                                        "automatically when it's initialized and end the session " +
+                                                        "when it's closed."),
+                Options.AutoSessionTracking);
+
+            Options.AutoSessionTrackingInterval = EditorGUILayout.IntField(
+                new GUIContent("Session Timeout [ms]", "The duration of time a session can stay paused " +
+                                                       "before it's considered ended."),
+                Options.AutoSessionTrackingInterval);
+
+            EditorGUILayout.EndToggleGroup();
+        }
+
+        private void DisplayDebug()
+        {
             Options.Debug = EditorGUILayout.BeginToggleGroup(
-                new GUIContent("Debug Mode", "Whether the Sentry SDK should print its diagnostic logs to the console."),
+                new GUIContent("Enable Debug Output", "Whether the Sentry SDK should print its " +
+                                                      "diagnostic logs to the console."),
                 Options.Debug);
+
             Options.DebugOnlyInEditor = EditorGUILayout.Toggle(
-                new GUIContent(
-                    "Only In Editor",
-                    "Only print logs when in the editor. Development builds of the player will not include Sentry's SDK diagnostics."),
+                new GUIContent("Only In Editor", "Only print logs when in the editor. Development " +
+                                                 "builds of the player will not include Sentry's SDK diagnostics."),
                 Options.DebugOnlyInEditor);
+
             Options.DiagnosticLevel = (SentryLevel)EditorGUILayout.EnumPopup(
-                new GUIContent("Verbosity level", "The minimum level allowed to be printed to the console. " +
+                new GUIContent("Verbosity Level", "The minimum level allowed to be printed to the console. " +
                                                   "Log messages with a level below this level are dropped."),
                 Options.DiagnosticLevel);
-            EditorGUILayout.EndToggleGroup();
 
             EditorGUILayout.EndToggleGroup();
-
-            // groupEnabled = EditorGUILayout.BeginToggleGroup("Sentry CLI Options", groupEnabled);
-            // uploadSymbols = EditorGUILayout.Toggle("Upload Proguard Mappings", uploadSymbols);
-            // auth = EditorGUILayout.TextField("Auth token", auth);
-            // organization = EditorGUILayout.TextField("Organization", organization);
-            // project = EditorGUILayout.TextField("Project", project);
-            // EditorGUILayout.EndToggleGroup();
         }
 
         private void OnLostFocus()
         {
+            if (Options is null)
+            {
+                return;
+            }
+
+            if (!AssetDatabase.IsValidFolder(ScriptableSentryUnityOptions.GetConfigPath(SentryOptionsAssetName)))
+            {
+                return;
```

Comment:
When would this happen? Should we use the diagnostic logger here to let us know?

Reply:
I kept crashing Unity when I deleted the config while the window was still open and then closing it.

Reply:
I see. Should we add that as a comment? I say that because just by eye balling the code it seemed redundant